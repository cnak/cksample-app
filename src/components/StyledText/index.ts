import { Platform } from 'react-native';
import styled, { css } from 'styled-components/native';

import { TEXT_COLOUR } from '../colors';

export default styled.Text`
    color: ${TEXT_COLOUR};
    font-family: Helvetica;
    ${Platform.select({
        ios: css`
            font-family: "Helvetica";
        `,
        android: css`
            font-family: Roboto;
        `
    })};
    font-size: 16px;
`;
