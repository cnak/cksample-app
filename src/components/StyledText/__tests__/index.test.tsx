import React from 'react';
import renderer from 'react-test-renderer';

import StyledText from '..';

describe('<StyledText>', () => {
    it('matches snapshot', () => {
        const tree = renderer.create(<StyledText>This is a test</StyledText>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
