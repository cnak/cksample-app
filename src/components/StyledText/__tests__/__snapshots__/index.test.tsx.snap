// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledText> matches snapshot 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#0B0C0C",
        "fontFamily": "Helvetica",
        "fontSize": 16,
      },
    ]
  }
>
  This is a test
</Text>
`;
